docs/src/architectures/nanopet.rst

```diff
@@ -111,3 +111,6 @@ The hyperparameters for training are
   to the RMSE
 :param loss: The loss function to use, with the subfields described in the previous
   section
+:param best_model_metric: specifies the validation set metric to use to select the best
+    model. The default is ``rmse_prod``, i.e., the product of the RMSEs for each target.
+    Other options are ``mae_prod`` and ``loss``.
```

Comment:
Do we define a concept of "best model" somewhere? Saying that this is the model that will end up in the cwd, named xxx.pt, …?

I would then link from here to this best model definition. Otherwise I would add something here, like "i.e. the model you can find at xxx.pt at the end of a training run"

Reply:
This can change for different architectures, so I will add it here